src/transformers/generation_flax_utils.py

```diff
@@ -209,8 +211,11 @@ def generate(
 
             input_ids (`jnp.ndarray` of shape `(batch_size, sequence_length)`):
                 The sequence used as a prompt for the generation.
-            max_length (`int`, *optional*, defaults to 20):
-                The maximum length of the sequence to be generated.
+            max_length (`int`, *optional*, defaults to `model.config.max_length`):
+                The maximum length of the sequence to be generated. Prefer the use of `max_new_tokens`, which ignores
+                the number of tokens in the prompt.
```

Comment:
Note that this is also the case for `max_length` if the model is an encoder-decoder (that the prompt length is ignored). For encoder-decoder models `max_length = max_new_tokens + 1` holds almost always true. Maybe
```suggestion
                 The maximum length the generated tokens can have corresponds to the length of the input prompt + `max_new_tokens`. In general, prefer the use of `max_new_tokens`, which ignores
```